scripts/templates/ldrddi.cpp.mako

```diff
@@ -272,9 +272,14 @@ namespace ur_loader
         %>
         %for i, item in enumerate(epilogue):
         %if 0 == i and not item['release']:
-        if( ${X}_RESULT_SUCCESS != result )
+        %if re.match(r"urEnqueue\w+", th.make_func_name(n, tags, obj)):
+        if( ${X}_RESULT_SUCCESS != result && ${X}_RESULT_ERRPR_ADAPTER_SPECIFIC != result )
```

Comment:
my typo from the chat
```suggestion
        if( ${X}_RESULT_SUCCESS != result && ${X}_RESULT_ERROR_ADAPTER_SPECIFIC != result )
```

also I think it's worth adding a comment here to the effect of "In the event of ERROR_ADAPTER_SPECIFIC we should still attempt to wrap any output handles below"